Pipfile

```diff
@@ -8,19 +8,22 @@ lint = "flake8 ."
 publish = "python setup.py sdist upload"
 test = "nosetests"
 type_check = "mypy ."
+format = "yapf -ir deadbolt tests"
+format_check = "yapf -dr deadbolt tests"
 
 [dev-packages]
-"flake8-debugger" = "*"
-"flake8-quotes" = "*"
-"flake8" = "*"
-"pep8-naming" = "*"
+flake8-debugger = "*"
+flake8-quotes = "*"
+flake8 = "*"
+pep8-naming = "*"
 coverage = "*"
 nose = "*"
 rednose = "*"
 sure = "*"
-Flask = "*"
+flask = "*"
```

Comment:
I just realized this was wrongly introducesd as a `dev-package` when it should be in the `install_requires` section of `setup.py`. Flask is necessary for this package to work (or at least the `FlaskRelayer` class).